### DISCUSSION TITLE
A Pinhole Camera Effect - BTW:

### DISCUSSION THREAD

**Discussion by flowerhorse** [brunoherbelin/vimix#85] [Ideas]
For a long time I have had the idea of a pinhole effect, which simulates a film exposing  to a single pinhole camera photo still over the film length like it could be seen if the process wouldn't be in the dark.
While the film is running, the individual frames are added together as in a simulated exposure in Auto ISO over the duration of the film. The result would be a non-moving pinhole camera photo which gradually overlaps all movements. The reverse way (a film is recreated from a pinhole photo) could also be interesting.
According to the logarithmic rules the exposure the visual change would take place in large periods of time. Therefore it could be used as a basic element of the session to which you can flash back in between.

**Comment by brunoherbelin** [comment]
do you mean something like this ?

https://user-images.githubusercontent.com/60393189/231857279-cda3d268-b836-46fd-8128-05efd915989c.mp4

**Reply by brunoherbelin** [reply]
This is obtained with a loopback source in front, with 'Hard light' Blending mode.
